lib/redux/home/home_state.dart

```diff
@@ -28,7 +28,7 @@ class HomeState {
   // Copies the states of the app or replaces for new ones if needed.
   HomeState copyWith(
       {LoadingStatus loadingStatus,
-      MyPlotState myPlotState,
+//      MyPlotState myPlotState,
```

Comment:
This will likely generate merge conflicts as i deleted it on my branch.
Doesnt matters but, we need to keep in mind that this line should disappear :)